### PR TITLE
feat: move actions interface from v1 to v2

### PR DESCRIPTION
**Pull request by kilosonc**

### CODE REVIEW

File: src/pages/clusters/Pods/components/RolloutDeployPanel.tsx [hunk 1]
```diff
@@ -227,6 +228,18 @@ function DeployButtons({
         >
           {intl.formatMessage({ id: 'pages.pods.deployAll' })}
         </Button>
+        <Button
```
Comment (bysph) [inline src/pages/clusters/Pods/components/RolloutDeployPanel.tsx]: Should the button be switched to "Cancel Auto Deploy" when entering automatic deploying mode?

File: src/locales/zh-CN/pages.ts [hunk 2]
```diff
@@ -89,9 +89,14 @@ export default {
   'pages.pods.deployStep': '发布阶段',
   'pages.pods.manualPause': '人工暂停',
   'pages.pods.deployAll': '全部发布',
+  'pages.pods.autoDeploy': '自动分批发布',
+  'pages.pods.cancelAutoDeploy': '取消自动分批发布',
   'pages.pods.deployCancel': '取消发布',
   'pages.pods.unpause': '取消暂停',
   'pages.pods.buildLog': '构建日志',
+  'pages.message.cluster.autoDeploy.success': '配置自动发布',
```
Comment (ryanwuer) [inline src/locales/zh-CN/pages.ts]: 配置自动发布成功？

**Review by bysph — APPROVED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -4,7 +4,7 @@ WORKDIR /app
 
 COPY . /app/
 
-RUN yarn && yarn build
+RUN yarn install && yarn umi build
```
Comment (ryanwuer) [inline Dockerfile]: Docker build will fail if i change base image from `node:14.21-alpine` to `node:18.16.0-alpine3.17`.

File: src/locales/zh-CN/pages.ts (hunk 2 above)
Comment (kilosonc) [reply src/locales/zh-CN/pages.ts]: fixed

File: Dockerfile (hunk 3 above)
Comment (kilosonc) [reply Dockerfile]: fixed

**Pull request merged**